test/unittest/tests/wrappers/config_parser.cpp

```diff
@@ -0,0 +1,37 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+#include "config_parser.h"
+#include <string>
+
+config_parser::config_parser(std::map<std::string, std::string> map) : _config_map(map)
+{
+    _cfg[0] = _cfg[1] = _cfg[2] = nullptr;
+}
+
+std::map<std::string, std::string> &
+config_parser::get_config_map()
+{
+    return _config_map;
+}
```

Comment:
Adding a `const` version of `get_config_map` allows us to pass configs as `const&` parameters in more places (the header needs changing too)

std::map<std::string, std::string> const&
config_parser::get_config_map() const
{
    return _config_map;
}